README.md

```diff
@@ -29,51 +29,51 @@ VPF stands for Video Processing Framework. It’s set of C++ libraries and Pytho
-  - Alternatively you can download VPF source code and build it from source by following the instructions provided. 
+  VPF provides python bindings over C++ API’s on top of Video Codec SDK.  Compiling VPF source code builds VPF binary (WHL file of PyNvCodec python module).
+  “pip install” automatically downloads source code and builds VPF binaries on client machines. This is quick and is the recommended way. 
+  Alternatively you can download VPF source code and build it from source by following the instructions provided. 
 
   - ### Installation using pip command
     - ### Installing using pip command on Windows
-    - Indicate path to your FFMPEG installation (with subfolders `bin` with DLLs, `include`, `lib`) in “TC_FFMPEG_ROOT” environment variable
-    - Run the following command.
-    ```
-    set SKBUILD_CONFIGURE_OPTIONS="-DTC_FFMPEG_ROOT=C:/path/to/your/ffmpeg/installation/ffmpeg/"
-    ```
-    - Run the following command.
-    ```
-    pip3 install git+https://github.com/NVIDIA/VideoProcessingFramework
-    ``` 
-    - If you have already cloned this repository just run 
-    ```
-    pip3 install .
-    ```
+      - Indicate path to your FFMPEG installation (with subfolders `bin` with DLLs, `include`, `lib`) in “TC_FFMPEG_ROOT” environment variable
+      - Run the following command.
+      ```
+      set SKBUILD_CONFIGURE_OPTIONS="-DTC_FFMPEG_ROOT=C:/path/to/your/ffmpeg/installation/ffmpeg/"
+      ```
+      - Run the following command.
+      ```
+      pip3 install git+https://github.com/NVIDIA/VideoProcessingFramework
+      ``` 
+      - If you have already cloned this repository just run 
+      ```
+      pip3 install .
+      ```
 
-  - ### Installing using pip command on Ubuntu 20.04 and Ubuntu 22.04
-    - Run the command below to install and update dependencies
-    ```
-    apt install -y  libavfilter-dev \ libavformat-dev \ libavcodec-dev \ libswresample-dev \ libavutil-dev\ wget \ 	cmake \ build-essential \    	git \      	libnvidia-encode-XXX \ libnvidia-decode-XXX
-    ```
-    - Install CUDA Toolkit 11.8 and above (if not already present)
-    - Ensure that nvcc location is mentioned in $PATH (most commonly already done by the CUDA installation)
-    ```
-    $ export PATH=/usr/local/cuda/bin:$PATH
-    ```
-  - Run the following command.
-    ```
-    pip3 install git+https://github.com/NVIDIA/VideoProcessingFramework
-    ``` 
-    - If you have already cloned this repository just run 
-    ```
-    pip3 install .
-    ```
+    - ### Installing using pip command on Ubuntu 20.04 and Ubuntu 22.04
+      - Run the command below to install and update dependencies
+      ```
+      apt install -y  libavfilter-dev \ libavformat-dev \ libavcodec-dev \ libswresample-dev \ libavutil-dev\ wget \ 	cmake \ build-essential \    	git \      	libnvidia-encode-XXX \ libnvidia-decode-XXX
```

Comment:
this needs to be line-by-line with `\` as it was before or in one line without the `\`